src/CMakeLists.txt

```diff
@@ -85,6 +96,13 @@ SET_TARGET_PROPERTIES(
     VERSION ${CLIENT_VERSION}
     SOVERSION ${PAHO_VERSION_MAJOR}
     DEFINE_SYMBOL "MQTT_EXPORTS")
+
+GENERATE_EXPORT_HEADER(paho-mqtt3a
```

Comment:
If we generate the export declarations header, we should remove the manually written one.
But this will break build via make (@icraggs )

Reply:
Oh, yes, you're right @waltronix. I had forgotten about the make build in this repo.